packages/typespec-client-generator-core/src/context.ts

```diff
@@ -38,12 +38,11 @@ export interface TCGCEmitterOptions extends SdkEmitterOptions {
   "emitter-name"?: string;
 }
 
-export interface SdkEmitterOptions {
+export interface UnbrandedSdkEmitterOptions {
   "generate-protocol-methods"?: boolean;
   "generate-convenience-methods"?: boolean;
   "api-version"?: string;
   "examples-dir"?: string;
```

Comment:
For Java, currently we only expose `license` emitter option. What about .NET? Do they need to expose `generate-protocol-methods` `generate-convenience-methods`?

For input from Srikanta as well.

Reply:
we are going to export each option separately which could give emitter enough flexibility.